optional/radicale/Dockerfile

```diff
@@ -2,13 +2,13 @@ ARG DISTRO=alpine:3.10
 FROM $DISTRO
 
 RUN apk add --no-cache curl bash python3 \
- && pip3 install radicale==2.1.12
+ && pip3 install radicale
```

Comment:
could we still pin this, at least to a major version, so we don’t get into surprise-failing-builds again?